src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -196,9 +196,8 @@ protected internal virtual void ValidateBinding(BindingMetadata bindingMetadata)
 
         protected static void ApplyMethodLevelAttributes(FunctionMetadata functionMetadata, BindingMetadata triggerMetadata, Collection<CustomAttributeBuilder> methodAttributes)
         {
-            if (functionMetadata.IsDisabled ||
```

Comment:
Add IsDisabled check back here?

Reply:
This check was for "isDisabled" defined in "function.json". We do not need to add "NoAutomatic" attribute for triggers as now "core" reads app settings and handles disabling.

Reply:
Ah, right